README.md

```diff
@@ -613,51 +615,97 @@ There are many options for training the model with your synthetic data-enhanced
 
 > **Note:** **Every** `ilab` command needs to run from within your Python virtual environment.
 
-#### Train the model locally on Linux
+#### ilab model training pipelines
+
+`ilab model train` has three pipelines: `simple`, `full`, and `accelerated`. The default is `accelerated`
+
+1. `simple` uses an SFT Trainer on Linux and MLX on MacOS. This type of training takes roughly an hour and produces the lowest fidelity model but should indicate if your data is being picked up by the training process.
+2. `full` uses a custom training loop and data processing funcions for the granite family of models. This loop is optimizied for CPU an MacOS functionality. Please use `--pipeline=full` in combination with `--device=cpu` (Linux) or `--device=mps` (MacOS). You can also use `--device=cpu` on a MacOS machine. However, MPS is optimized for better performance on these systems.
```

Comment:
typo? CPU on MacOS

Reply:
cpu will "work" on Mac, but I don't see why people would use it since memory is unified and MPS is better, but worth noting that it will run I think?

Reply:
oh I see the typo now :facepalm: sorry

Reply:
fixed to: `CPU and MPS`